src/wp-includes/option.php

```diff
@@ -77,7 +77,10 @@
 function get_option( $option, $default = false ) {
 	global $wpdb;
 
-	$option = trim( $option );
+	if ( is_string( $option ) ) {
```

Comment:
Need to change to `is_scalar()` to ensure floats are casted to int to ensure no b/c break if a float is passed in as the option. Also added more tests during live working session. Update is coming.

Reply:
PR has been updated with all changes discussed during the livestream. PR description has been updated to match.

Reply:
Ready for commit AFAIC.